src/reader/reader_utils.c

```diff
@@ -31,8 +31,6 @@ t_bool	reader_is_special(char *str)
 	}
 	if (PIPE_CHAR == *str)
 		return (1);
-	if (SEMICOLON_CHAR == *str)
```

Comment:
I am not sure about this one

Reply:
We added it at the beggining of the project before knowing we didnt have to implement it, so i guess it is useless now. In fact it was probably a bug now